### ISSUE TITLE
Hardcoded language grammar path in PrismJS autoloader config results in 404s

### ISSUE THREAD

**Issue by colinwilson** [colinwilson/lotusdocs#78]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for support

### Describe the bug

See discussion https://github.com/colinwilson/lotusdocs/discussions/77#discussioncomment-7035028

The hardcoded [PrismJS autoloader plugin](https://prismjs.com/plugins/autoloader/) language path causes `404` errors if a Hugo project is deployed in a path other than root.

### Expected behavior

The PrismJS autoloader should find language grammars regardless of the site's `baseURL`

### Environment

```markdown
- OS: Windows 11
- Browser: FireFox / Chrome
- Browser Version: 116
```

**Comment by colinwilson** [comment]
Fixed by a6d0d9b

**Edit**: Does not work! PrismJS looks for the language grammar files relative to the current path instead of the actual path.

**Comment by colinwilson** [comment]
Fixed by https://github.com/colinwilson/lotusdocs/commit/bc1752e7395a7312711de50effd782fb58ecc225
